### ISSUE TITLE
Lab 05 Code

### ISSUE THREAD

**Issue by dshartmann** [DS4PS/cpp-524-sum-2021#23]
I ran into this error trying to run the R code provided in the Lab Instructions. I see the problem with the code is the source, but I'm not sure how to fix it. 

<img width="810" alt="Screen Shot 2021-06-16 at 11 04 34 AM" src="https://user-images.githubusercontent.com/77824972/122270892-a9402600-ce93-11eb-8050-67703d53e2c5.png">

Also, I'm not quite sure where to place/run this chunk? I thought it might fix the above problem but I think I might be coding it wrong.

<img width="596" alt="Screen Shot 2021-06-16 at 11 12 53 AM" src="https://user-images.githubusercontent.com/77824972/122271031-d12f8980-ce93-11eb-9862-480b8c7c5ab3.png">

**Comment by Dselby86** [comment]
Both of those commands are reading data from your file location.  Do you
have a file named what is in the string?

Is that file saved in the same location as the .rmd file?

**Comment by dshartmann** [comment]
I don't. I don't think we have used data in that format before, only URL. Is there a file with the data we can download, and where do we find it? I'm only seeing the "data/np-ceo-comp-data.csv" and "diff_n_diff_graph4.R"

**Comment by krittschof** [comment]
What are the directions for Lab 5? I am unsure what to do. Please advise here or preferably to my inquiry yesterday.  I am also unable to put the code in R and run it...thinking we just needed to recreate the Lab 5 information. I even went to Github to find the repository, which I think I found, but am unable to load it to R.  Please help.
Kerri

**Comment by dshartmann** [comment]
@krittschof I just assumed we have to create an RMD like we did for Lab 02, based off of the Lab 05 instructions provided in the Course Schedule. The problem I'm running into is finding/loading the data. I haven't made it past the first two chunks of code so I don't even know what do to past that... It sounds like we might be having the same issue?

**Comment by krittschof** [comment]
@dshartmann Thank you for your reply. I do not see any instructions in the Course Schedule for Lab 5, I can totally be missing something. I read through the information twice to see if I was missing. But like you, I tried to run the information, just load the dataset and I also cant get passed the first chunk. Again, thank you!

**Comment by dshartmann** [comment]
This is what my Course Schedule looks like, is yours different?

<img width="947" alt="Screen Shot 2021-06-16 at 1 34 09 PM" src="https://user-images.githubusercontent.com/77824972/122289446-946d8d80-cea7-11eb-8e9a-97bda53149ec.png">

**Comment by krittschof** [comment]
Good call! That is what mine says too, and then when I clikc on the Lab-05 Instructions it is a scenario in which the information is already presented. So possibly we answer the question as noted above (thank you again for bringing to light) using the information in the instructions and responding in a Word document. I am just thinking out loud

**Comment by AmyTrumpower** [comment]
When I read the download from the Lab-05 Instructions button from above, the document seems to give the answer in the conclusion.

**Comment by dshartmann** [comment]
Oh maybe that is what we are supposed to do! That would make sense. Honestly I didn't read through all of the code section I just figured I would follow it as I put it into R. Hopefully we can receive confirmation soon.

**Comment by chmao99** [comment]
> When I read the download from the Lab-05 Instructions button from above, the document seems to give the answer in the conclusion.

Totally agree with you!

**Comment by Dselby86** [comment]
Sorry for the confusion everyone.

I'm working on posting the Lab instructions to the Canvas page.
Unfortunately GitHub is not cooperating with me.  Also going on the Canvas
page will be the datafile that you need to answer the lab.

**Comment by lecy** [comment]
Sorry for the confusion on the instructions. The goal here is to see what happens when the three estimators are applied to the same research question. It is an interpretative lab (you don't have to run the models yourselves). The goal is to (1) see that the estimators are not substitutes - they produce very different insights, (2) think about what each estimator might represent and how we would interpret the specific component, and (3) identify the best estimator for the specific research question (the discrimination component of the pay gap). 

Instructions have been updated to make this clear: 

--------

This final lab offers an opportunity to synthesize material on counterfactuals by calculating program impact in a couple of different ways with the same data set and thinking about which estimator (counterfactual) best suits the research question. 

You are presented with analysis that estimates the gender pay gap of nonprofit executive directors using three different models. The key insight is being able to determine which of the three estimators you are using in each regression model (reflexive, post-test only, or pre-post with comparison), what each estimator represents, and which estimate is most appropriate for your research question. 

<a class="uk-button uk-button-default" href="https://ds4ps.org/cpp-524-sum-2020/labs/lab-05-gender-wage-gap.html">Lab-05 Instructions</a>

  
**Instructions** 
 
You will see that the pay gap will change when you change the estimator. Explain in your own words what you believe each represents (see the diagram on components of the pay gap for ideas), then select the one that you believe is the best measure for the pay discrimination component of the gender pay gap (unequal pay for equal work). 

Write up your argument and submit via Canvas.

You do not need to run models to answer the questions above, but if you would like to follow along you can load the data and auxiliary functions as follows: 
  
```r
source( "https://raw.githubusercontent.com/DS4PS/cpp-524-sum-2021/main/labs/diff_n_diff_graph4.R" )
URL <- "https://github.com/DS4PS/cpp-524-sum-2021/blob/main/labs/data/np-ceo-comp-data.csv?raw=true"
d <- read.csv( URL )
```

**Comment by lecy** [comment]
> When I read the download from the Lab-05 Instructions button from above, the document seems to give the answer in the conclusion.

It's a little reductive - there's no gotcha here, just a check to see if everything is making sense to you at this point and whether you can put it all in your own words.

**Comment by jacobtnyoung** [comment]
I am confused. Are we supposed to complete the lab per the instructions that were posted to the Lab 5 page on Canvas?
